app/src/main/java/com/checkin/app/checkin/home/model/NearbyRestaurantModel.kt

```diff
@@ -19,7 +19,9 @@ data class NearbyRestaurantModel(
         val distance: Double,
         val ratings: Double,
         val cuisines: List<String>,
-        val offer: RestaurantListingOfferModel?
+        val offer: RestaurantListingOfferModel?,
+        val is_open: Boolean,
```

Comment:
Variables in Java/Kotlin aren't named in snake_case. Even if the API requires this key, you should use @JsonProperty annotation here and keep the variable name proper